templates/admin/change_form.html

```diff
@@ -1,5 +1,6 @@
 {% extends 'admin/change_form.html' %}
 {% load static %}
+{% load i18n admin_urls %}
```

Comment:
Are we using `i18n` anywhere? I think that's for translation.

```suggestion
{% load admin_urls %}
```